### ISSUE TITLE
Did something changed?  It stopped working

### ISSUE THREAD

**Issue by tfer6** [zachowj/xfinity-data-usage#53]
Hi,  
Been running this for 3-4 months now without really any issue.  All of the sudden this week it just up and stopped being able to login and timing out.  I tried creating with a fresh container and no luck.  I was able to login directly to comcast so I don't think it's my username/pw

Here's what I'm seeing on the log.  Let me know if there is anything else I can provide to help

```
today at 2:32:36 PM2021-12-11 14:32:36 [info] : Xfinity Update every 20 mins
today at 2:32:36 PM2021-12-11 14:32:36 [info] : Http server will be started
today at 2:32:36 PM2021-12-11 14:32:36 [info] : Will publish to MQTT (Home Assistant) on new data
today at 2:32:36 PM2021-12-11 14:32:36 [info] : http server started
today at 2:32:36 PM2021-12-11 14:32:36 [error] : Unable to load password file.
today at 2:32:36 PM2021-12-11 14:32:36 [verbose] : Fetching Data
today at 2:32:36 PM2021-12-11 14:32:36 [debug] : Loading (https://customer.xfinity.com)
today at 2:32:36 PM2021-12-11 14:32:36 [info] : Connected to MQTT
today at 2:32:45 PM2021-12-11 14:32:45 [debug] : Login redirects: 3
today at 2:32:45 PM2021-12-11 14:32:45 [debug] : Logging in
today at 2:33:16 PM2021-12-11 14:33:16 [error] : TimeoutError: Navigation timeout of 30000 ms exceeded. Next update at 2:52:36 PM
```

**Comment by janssenjones** [comment]
Yes, this was due to a recent change with the Comcast /Xfinity website, which required an update to the container to resolve.  If you pull 17.1 release, it should fix it.  I don't think 17 or earlier will work any more.

**Comment by tfer6** [comment]
hmmm... strange.  I'm glad you pointed out that there was a 17.1.  I must have mistakenly updated this container.  I actually just rolled it back to 17.0 and it works just fine.  17.1 was the one producing the error I listed above.  I guess I'll consider this closed and hold on 17.0 until it our friends at comcast make trouble.

**Comment by janssenjones** [comment]
Yes, I had moved to 17.1 and it broke mine a week or so ago, so I rolled back.  Then Comcast updated their site for my login and I had to roll forward.  Ideally, there'd be a 17.2 that could handle either scenario, but it sounds like for now that folks will need to watch for their plugin to stop updating and then change accordingly.
